gulpfile.js

```diff
@@ -410,3 +411,33 @@ gulp.task('prod', function(callback) {
 gulp.task('dev:runserver', function(callback) {
     runSequence('dev', 'connect', 'watch', 'test:watch', callback);
 });
+
+//Image Optimization
+var imagemin = require('gulp-imagemin');
+var cache = require('gulp-cache');
+ gulp.task('images', function() {
+  return gulp.src('frontend/src/images/**/*')
+    .pipe(cache(imagemin({ optimizationLevel: 5, progressive: true, interlaced: true })))
+    .pipe(gulp.dest('frontend/src/images'));
+});
+
+//Image Optimization of Organization Folder
+var imagemin = require('gulp-imagemin');
+var cache = require('gulp-cache');
+ gulp.task('imagesorgan', function() {
+  return gulp.src('frontend/src/images/organizations/**/*')
+    .pipe(cache(imagemin({ optimizationLevel: 5, progressive: true, interlaced: true })))
+    .pipe(gulp.dest('frontend/src/images/organizations'));
+});
+
+ // Concatenate & Minify JS
+gulp.task('scripts', function() {
+    return gulp.src('frontend/src/js/controllers/*.js')
+      .pipe(concat('controllers.js'))
+        .pipe(rename({suffix: '.min'}))
+        .pipe(uglify())
+        .pipe(gulp.dest('frontend/src/js/controllers'));
+});
+// Default Task
+gulp.task('default', ['scripts', 'images', 'imagesorgan']);
+
```

Comment:
Add minified files only to production task, please let me know if you need any help.

Reply:
@gautamjajoo ^^